streams/src/test/java/org/apache/kafka/test/MockInternalProcessorContext.java

```diff
@@ -55,14 +55,9 @@ public MockInternalProcessorContext(final Properties config, final TaskId taskId
         super(config, taskId, stateDir);
     }
 
-    @Override
-    public void setSystemTimeMs(long timeMs) {
```

Comment:
The purpose of this class seems to be to allow mocking the time -- given that we add proper time support now, I am wondering if we can actually delete the whole class?

Reply:
@mjsax should i try to do this in another pr?

Reply:
I guess a follow up PR works, too.